ratingTypes.ts

```diff
@@ -20,20 +20,25 @@ export interface Rating {
     dateCreated: Date;
     dateEdited?: Date;
 }
- 
-export interface Place {
-	_id: GooglePlace["place_id"];
-	guideDogAvg: number | null;
-	isMenuAccessibleAvg: number | null;
-	noiseLevelAvg: number | null;
-	lightingAvg: number | null;
-	isStaffHelpfulAvg: number | null;
-	isBathroomOnEntranceFloorAvg: number | null;
-	isContactlessPaymentOfferedAvg: number | null;
-	isStairsRequiredAvg: number | null;
-	spacingAvg: number | null;
-	promotion: {
-		monthly_budget: number;
-		max_cpc: number;
-	};
+
+export const fieldsToNamesMap: Record<string,string> = {
+    avgGuideDogFriendly: "Guide Dog Friendliness",
+    isMenuAccessibleAvg: "Menu Accessibility",
+    noiseLevelAvg: "Noise Level",
+    lightingAvg: "Lighting",
+    isStaffHelpfulAvg: "Staff Helpfulness",
+    isBathroomOnEntranceFloorAvg: "Bathroom On Entrance Floor",
+    isContactlessPaymentOfferedAvg: "Contactless Payment Offered",
+    isStairsRequiredAvg: "Stairs Required",
+    spacingAvg: "Spacing",
+  };
+
+function buildNamesToFieldsMap(){
```

Comment:
What is this for? Can we add comments to these functions actually?

Reply:
Sure thing